src/state/sagas/scenario/PollScenarioState/PollScenarioState.js

```diff
@@ -31,8 +34,17 @@ export function* pollScenarioState(action) {
         // Update the scenario state in all scenario redux states
         yield put({
           type: SCENARIO_ACTIONS_KEY.UPDATE_SCENARIO,
-          data: { scenarioState: data.state, scenarioId: action.scenarioId, lastRun: data.lastRun },
+          data: {
+            scenarioState: data.state,
+            scenarioId: action.scenarioId,
+            lastRun: data.lastRun,
+          },
         });
+        if (action.startTime != null) {
```

Comment:
Could be interesting to change the condition by `( action.startTime )` to manage truthy values.
See https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in